docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst

```diff
@@ -4,7 +4,282 @@
 Bareos Webui
 ============
 
-This chapter addresses the usage of the |webui|.
+.. _section-webui-command-acl-requirements:
+
+Command ACL Requirements
+------------------------
+
```

Comment:
Why is this in Introduction And Tutorial? 
Who about moving it the Task and Concepts or Appendix?

Reply:
At the moment I would like to keep it as it is, but we should discuss how we would like to arrange topics in near future to improve the structure of the documentation.